conf/test.config

```diff
@@ -70,18 +70,9 @@ process {
         }
     }
 
-    if (params.joint_mutect2) {
-        withName: 'MUTECT2_PAIRED' {
-            ext.args         = { params.ignore_soft_clipped_bases ?
-                                    "--dont-use-soft-clipped-bases true --f1r2-tar-gz ${task.ext.prefix}.f1r2.tar.gz --normal-sample ${meta.normal_id}" :
-                                    "--f1r2-tar-gz ${task.ext.prefix}.f1r2.tar.gz --normal-sample ${meta.normal_id}" }
-        }
-    }
-    else {
-        withName: 'MUTECT2_PAIRED'{
-            //sample name from when the test data was generated
-            ext.args = { "--f1r2-tar-gz ${task.ext.prefix}.f1r2.tar.gz --normal-sample normal " }
-        }
+    withName: '.*:MUTECT2_PAIRED'{
```

Comment:
this is from the other open mutect2 PR?

Reply:
Yes, I plagiarised to get the tests working in my local.

Reply:
I was hoping that the other PR would be merged first and this would become a simple merge conflict on this PR.